readme.txt

```diff
@@ -46,13 +48,14 @@ Or you can follow these steps to install the TaxJar plugin:
 1. Install the plugin in your WordPress admin panel under *Plugins > Add New* (search for “TaxJar”) or [download the zip](https://downloads.wordpress.org/plugin/taxjar-simplified-taxes-for-woocommerce.zip) and upload to `/wp-content/plugins` on your server.
 2. Activate the plugin under *Plugins > Installed Plugins*.
 3. Find the TaxJar configuration under *WooCommerce > TaxJar*.
-4. You’ll need a TaxJar API token to get started. If you already have a TaxJar account, [click here](https://app.taxjar.com/account#api-access) to get an API token. In the TaxJar app, generate a new API token and copy it to your clipboard. If you don’t have a TaxJar account, you’ll be asked to sign up first. After signing up, follow the steps above.
-5. Paste your TaxJar API Token into the “API Token” field.
-6. Fill out the rest of your settings. TaxJar requires your city and zip code from which you ship products to calculate sales tax. We automatically detect your country and state based on your WooCommerce configuration.
-7. If you have multiple [nexus](https://blog.taxjar.com/sales-tax-nexus-definition/) states where you need to collect sales tax, make sure they're [added to your TaxJar account](https://app.taxjar.com/account#states). Click the “Sync Nexus Addresses” button to import your nexus addresses into WooCommerce. 
-8. Check the box next to “Enable TaxJar Calculations”.
-9. If you plan to use TaxJar for sales tax reporting and filing, check the box next to "Enable order downloads to TaxJar" for TaxJar to connect and download the transactions on your store for [AutoFile](https://www.taxjar.com/autofile/) and reporting features.
-10. Click “Save changes”. You're now up and running with TaxJar!
+4. Click the "Connect To TaxJar" button.
+5. Sign in with your TaxJar account (or create a new one) in the popup.
+6. Click the "Connect" button. The popup will automatically close and your WooCommerce store will be linked to your TaxJar account. Note: If your business exceeds 1,000 orders per month, you will be prompted to upgrade your account to TaxJar Professional, in order to activate real-time sales tax calculations through the TaxJar API.
```

Comment:
Minor nit: can we drop the comma after `Professional`?

Reply:
Thanks, fixed!